py-polars/tests/unit/test_sql.py

```diff
@@ -113,6 +113,225 @@ def test_sql_equal_not_equal() -> None:
     }
 
 
+def test_sql_trig() -> None:
+    df = pl.DataFrame(
+        {
+            "a": [-4, -3, -2, -1.00001, 0, 1.00001, 2, 3, 4],
+        }
+    )
+
+    c = pl.SQLContext(df=df)
+    res = c.execute(
+        """
+        SELECT
+        asin(1.0)/a as "pi values",
+        cos(asin(1.0)/a) AS "cos",
+        cot(asin(1.0)/a) AS "cot",
+        sin(asin(1.0)/a) AS "sin",
+        tan(asin(1.0)/a) AS "tan",
+
+        cosd(asind(1.0)/a) AS "cosd",
+        cotd(asind(1.0)/a) AS "cotd",
+        sind(asind(1.0)/a) AS "sind",
+        tand(asind(1.0)/a) AS "tand",
+
+        1.0/a as "inverse pi values",
+        acos(1.0/a) AS "acos",
+        asin(1.0/a) AS "asin",
+        atan(1.0/a) AS "atan",
+
+        acosd(1.0/a) AS "acosd",
+        asind(1.0/a) AS "asind",
+        atand(1.0/a) AS "atand"
+        FROM df
+        """,
+        eager=True,
+    )
+
+    df_result = pl.DataFrame(
+        {
+            "pi values": [
```

Comment:
`make pre-commit` makes this super long formatting, not sure if anyone has a good remedy for this.

Reply:
leave it, as long as it is automatic. :)